documentation/book/con-cluster-operator-rbac.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// assembly-operators-cluster-operator.adoc
+
+[id='con-cluster-operator-rbac-{context}']
+= Role-Based Access Control (RBAC)
+
+== Provisioning Role-Based Access Control (RBAC) for the Cluster Operator
+
+For the Cluster Operator to function it needs permission within the {ProductPlatformName} cluster to interact with the resources it manages and interacts with.
+These include the desired resources, such as  `Kafka`, `Kafka Connect`, and so on, as well as the managed resources, such as `ConfigMaps`, `Pods`, `Deployments`, `StatefulSets`, `Services`, and so on.
+Such permission is described in terms of {ProductPlatformName} role-based access control (RBAC) resources:
+
+* `ServiceAccount`,
+* `Role` and `ClusterRole`,
+* `RoleBinding` and `ClusterRoleBinding`.
+
+In addition to running under its own `ServiceAccount` with a `ClusterRoleBinding`, the Cluster Operator manages some RBAC resources because some of the components need access to {ProductPlatformName} resources.
+
+{ProductPlatformName} also includes privilege escalation protections that prevent components operating under one `ServiceAccount` from granting other `ServiceAccounts` privileges that the granting `ServiceAccount` does not have.
+Because the Cluster Operator must be able to create the `ClusterRoleBindings` and `RoleBindings` needed by resources it manages, the Cluster Operator must also have those same privileges.
+
+[id='delegated-privileges-{context}']
+== Delegated privileges
+
+When the Cluster Operator deploys resources for a desired `Kafka` resource it also creates `ServiceAccounts`, `RoleBindings` and `ClusterRoleBindings`, as follows:
+
+* The Kafka broker pods use a `ServiceAccount` called `<cluster-name>-kafka` (where `<cluster-name>` is a placeholder for the name of the `Kafka` resource)
+  - When the rack feature is used, the `strimzi-<cluster-name>-kafka-init` `ClusterRoleBinding` is used to grant this `ServiceAccount` access to the nodes within the cluster via a `ClusterRole` called `strimzi-kafka-broker`
+  - When the rack feature is not used no binding is created.
+* The Zookeeper pods use the default `ServiceAccount` as they have no need to access {ProductPlatformName} resources
+* The Topic Operator pod uses a `ServiceAccount` called `<cluster-name>-topic-operator` (where `<cluster-name>` is a placeholder for the name of the `Kafka` resource)
```

Comment:
called `<cluster-name>-topic-operator` (where `<cluster-name>` is a placeholder for the name of the `Kafka` resource)

TO

called `_<cluster-name>_-topic-operator`